src/index.css

```diff
@@ -1,27 +1,60 @@
-body {
-  margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    "Segoe UI",
-    "Roboto",
-    "Oxygen",
-    "Ubuntu",
-    "Cantarell",
-    "Fira Sans",
-    "Droid Sans",
-    "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+.myCalculator {
+  display: grid;
+  width: 35vw;
+  height: 100vh;
+  position: relative;
+  margin: 5% 45% 0 35%;
+  gap: 2px;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    "Courier New",
-    monospace;
+.special {
+  background-color: #ff9500;
+}
+
+.calculatorScreen {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  background-color: #505050;
+  color: whitesmoke;
+  height: 5vh;
+  font-size: 1.5rem;
+  padding: 2px 2px;
+  width: 94.2%;
+  margin-left: 2.56%;
+  margin-right: -4.8%;
+}
+
+/* grid buttons */
+
+.calculatorButtons {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  padding: 12px;
+  grid-row-gap: 0.6px;
+  grid-column-gap: 0.6px;
+  width: 100%;
+}
+
+.calculatorButtons > button {
+  text-align: center;
+  font-size: 1rem;
+  font-weight: 300;
+  text-transform: uppercase;
+  border: 1px solid #d4d4d2;
+  color: #000;
+}
+
+/* .calculatorButtons > button:hover {
+  background-color: #505050;
+} */
```

Comment:
- [ ] For best practices, kindly consider removing all unused commented code from your repo, it makes your code messy